app/build.gradle

```diff
@@ -73,6 +73,8 @@ dependencies {
     implementation 'androidx.room:room-runtime:2.2.6'
 
     implementation 'com.google.firebase:firebase-database:19.7.0'
+    implementation 'com.airbnb.android:lottie:3.7.0'
```

Comment:
Is it the required library to implement a splash screen? If interested, as an alternative, here is how I did it for another project last year: [splash screen class](https://github.com/Antho1426/capl-project/blob/master/capl-app/app/src/main/java/ch/epfl/mobots/capl/ui/SplashScreenActivity.java), [splash screen layout](https://github.com/Antho1426/capl-project/blob/master/capl-app/app/src/main/res/layout/activity_splash_screen.xml). Please don't judge my code ^^'. I'm a bit more experienced now (but still not perfect at all 😂), but at this time I had just no idea of organising and writing my code properly!

Reply:
It's a open source library with a lot of nice and free animation, I take it from a video

Reply:
Ok ok, didn't know this!